cmd/internal/config/config.go

```diff
@@ -19,28 +24,61 @@ type Keys struct {
 	PathToPublic  string `yaml:"public"`
 }
 
-type NodeConfig struct {
+type DegitxConfig struct {
 	Version   string             `yaml:"version"`
 	Keys      *Keys              `yaml:"keys"`
 	LogConfig *logging.LogConfig `yaml:"logging"`
 }
 
-func (config *NodeConfig) fromFile(fileName string) error {
+type errConfigNotFound struct{ paths []string }
+
+func (e *errConfigNotFound) Error() string {
+	return fmt.Sprintf("configuration file not found in {%s}",
+		strings.Join(e.paths, ":"))
+}
+
+func ParseConfig(ctx *cli.Context, preDeclaredPaths ...string) (*DegitxConfig, error) {
```

Comment:
@stepanvalyavskiy we still have the dependency on top level package from low level (see previous comments)

Reply:
@stepanvalyavskiy also, let's use similar names, e.g. `FromFiles(files ...string) error`